packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -126,6 +127,26 @@ const CommerceApiProvider = (props: CommerceApiProviderProps): ReactElement => {
             shopperSearch: new ShopperSearch(config),
             shopperSeo: new ShopperSeo(config)
         }
+        const isInStorefrontPreview = () => onClient() && window.STOREFRONT_PREVIEW
```

Comment:
Generally speaking I would like to see a solution where we don't leak implementation details directly about preview into this library. 

I believe I mentioned this previously, but if there is a way that we can set the cache breaker through the provider that would make for better separation of concerns.

Reply:
> if there is a way that we can set the cache breaker through the provider that would make for better separation of concerns

Does this mean having the provider accept a function as input instead? And the cache-breaking function is defined somewhere else (like in retail-react-app)?